src/cascadia/TerminalConnection/ConptyConnection.cpp

```diff
@@ -186,13 +186,15 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
                                        const HANDLE hRef,
                                        const HANDLE hServerProcess,
                                        const HANDLE hClientProcess,
-                                       TERMINAL_STARTUP_INFO startupInfo) :
+                                       const TERMINAL_STARTUP_INFO& startupInfo) :
         _rows{ 25 },
         _cols{ 80 },
         _guid{ Utils::CreateGuid() },
```

Comment:
HUH?

Reply:
Oh. I see. I see....
I fucked it up in my session restoration PR! 😄 That one moved the GUID over to the parent class, but I forgot to create the GUID here.

Reply:
Now that I think about it, it's surprising that the cherry-pick applied this cleanly.